pwnagotchi/plugins/default/AircrackOnly.py

```diff
@@ -28,15 +28,15 @@ def on_handshake(agent, filename, access_point, client_station):
     if result:
         logging.info("[AircrackOnly] contains handshake")
     else:
-        todetele = 1
+        todelete = 1
```

Comment:
This presumably significantly changes the behavior of the code. I believe this family of changes is the only significant change in this PR.